### PR TITLE
Add AddDisplayUploadAd example

### PR DESCRIPTION
**Pull request by devchas**

### CODE REVIEW

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddDisplayUploadAd.java [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+   * @throws GoogleAdsException if an API request failed with one or more service errors.
+   */
+  private void runExample(GoogleAdsClient googleAdsClient, long customerId, long adGroupId)
+      throws IOException {
+    try {
+      // There are several types of display upload ads. For this example, we will create
+      // an HTML5 upload ad, which requires a media bundle.
+      // The DisplayUploadProductType field lists the available display upload types:
+      // https://developers.google.com/google-ads/api/reference/rpc/v3/DisplayUploadAdInfo
+      // Creates a new media bundle asset and returns the resource name.
```
Comment (jradcliff) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddDisplayUploadAd.java:116]: nit: add a blank line to separate the paragraph above from this line comment.

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddDisplayUploadAd.java [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+      String adAssetResourceName = CreateMediaBundleAsset(googleAdsClient, customerId);
```
Comment (jradcliff) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddDisplayUploadAd.java:117]: Please use lowercase method names

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddDisplayUploadAd.java [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+
+      // Creates a new display upload ad and associates it with the specified ad group.
+      CreateDisplayUploadAdGroupAd(googleAdsClient, customerId, adGroupId, adAssetResourceName);
+    } catch (GoogleAdsException e) {
```
Comment (jradcliff) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddDisplayUploadAd.java:121]: No need for a try/catch here. The `main()` impl already has this.

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddDisplayUploadAd.java [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+    private Long customerId;
+
+    @Parameter(names = ArgumentNames.AD_GROUP_ID, required = true)
+    private Long adGroupId;
+  }
+
+  public static void main(String[] args) throws IOException {
+    AddDisplayUploadAdParams params = new AddDisplayUploadAdParams();
+    if (!params.parseArguments(args)) {
+
```
Comment (AndrewMBurke) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddDisplayUploadAd.java:65]: Nit: extra blank line between brace and comment; doesn't seem consistent with rest of code, e.g. the catch below on line 87.

Comment (devchas) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddDisplayUploadAd.java:65]: The extra line in this specific line is consistent with all of the other Java code examples.

**Pull request merged**